src/avitab/apps/MapApp.h

```diff
@@ -40,6 +44,83 @@
 
 namespace avitab {
 
+struct OnlineSlippyMapConfig {

```

Comment:
I think it would make sense to move this struct and its JSON deserializer code into its own header as to not clutter up the MapApp header too much. And since `from_json` is not a template, I don't think the implementation should live in the header file. So maybe it would make sense to introduce `OnlineSlippyMapConfig` as a custom module with a `cpp` and `h` file.

Reply:
Moved it in a new set of files OnlineSlippyMapConfig.h/cpp